File: 172 Factorial Trailing Zeroes/solution.cpp
class Solution {
public:
    int trailingZeroes(int n) {
        int count = 1;
        while (pow(5,count)<=n)
            count++;
        int result = 0;
        for (int i=1;i<count;i++){
            result+=n/(pow(5,i));
        }
        return result;
    }
};

File: 172 Factorial Trailing Zeroes/solution.py
class Solution:
    # @param {integer} n
    # @return {integer}
    def trailingZeroes(self, n):
        result,count = 0,0
        tmp = n
        while (tmp/5>=1):
            count+=1
            tmp = tmp/5
        for i in range(count):
            result+=int(n/math.pow(5,i+1))
        return result